app/src/interfaces/select-color/select-color.vue

```diff
@@ -406,14 +407,16 @@ function useColor() {
 	padding-right: 0px;
 }
 
-.v-input.html-color-select {
-	width: 0;
-	height: 0;
-	visibility: hidden;
-}
+.color-input {
+	:deep(.input) {
+		padding-left: 6px;
+	}
```

Comment:
Is this even needed anymore?

Reply:
Can confirm it is not needed anymore 👍 I was only trying to keep the original padding-left intact without noticing it's also not relevant anymore, thanks for catching this!